### DISCUSSION TITLE
Task1: 调研pyglet音频资源播放的问题

### DISCUSSION THREAD

**Discussion by alanshi** [py-pal/py-pal#1] [Tasks]
1: 在pyglet 2.0.10版本中mp3播放异常, 1.5.28播放正常
2: 暂时不确定如何在pyglet中播放mid 资源

**Comment by rexzhang** [comment]
不要在事件中播放即可,事件中播放只会播放头几秒 c32c73e613d6a1fa3889e8634efb8ce16f539a58
